Fix violations of Ruff rule S311

Phantominumm opened the pull request:
B311 and S311, using secrets instead of random() to generate number because it's easily predictable (crypto graphically weak)

https://docs.astral.sh/ruff/rules/suspicious-non-cryptographic-random-usage/

https://semgrep.dev/r?q=gitlab.bandit.B311

Commit:
B311 or S311 random generator

Commit:
Ruff rules pre-commit

fabcor-maxiv commented on mxcubeweb/routes/mockups.py:
```diff
@@ -91,27 +92,26 @@ def shape_mock_result(sid):
+                        int(secrets.randbelow(256)),
+                        int(secrets.randbelow(256)),
+                        0,  # int(random()) was used in the original code
+                        # but always returns 0
```
I do not understand this...

On one hand: there is no need to comment in the code what the code used to be doing, such explanation belongs in the commit message

On the other hand: do we need a zero there?

marcus-oscarsson replied:
Is true that the comment should be in a commit message. It was me who asked @Phantominumm to add a comment about it as the current logic was a bit odd. 

I was hoping that maybe one of you guys could answer because we don't use the RGB values for the mesh results. Could it be the alpha channel, but we though it was odd that it was always zero ?

As this is for some sort of mockup we did not want to change the logic, so we kept it as 0 for the time being.

fabcor-maxiv replied:
> I was hoping that maybe one of you guys could answer because we don't use the RGB values for the mesh results. Could it be the alpha channel, but we though it was odd that it was always zero ?

I'll try to get some feedback on this

fabcor-maxiv replied:
No feedback from the rest of the team (yet).

In principle, my impression is that having zeroes here is fair, since as said already the previous code would always return zeroes.

On the other hand: is this code still needed? I have not found a way to trigger this code in a "normal" way. I can of course call the HTTP URL explicitly, but even that fails.

marcus-oscarsson replied:
I think it was used by some test script that was triggered manually, or it was my interpretation at-least. I think we can remove the entire mockup route (actually I think it would be quite important that we did)

marcus-oscarsson replied:
If you agree, I suggest that we close this PR and we open a new one to remove the route ?

fabcor-maxiv replied:
I am okay with removing this route.

marcus-oscarsson commented:
Thanks

Phantominumm committed:
Remove comment

Previous version had int(random()) instead of 0 but the return was always 0 random() give a number < 1, was it intended or just forgotten ? change to just 0 for the moment while pending further explanations.
